### PR TITLE
Manage users in role/org forms

### PR DESCRIPTION
**Pull request by tonio**
See https://github.com/georchestra/georchestra/issues/2080

### CODE REVIEW

**Comment by fvanderbiest** [comment]
LGTM, thanks !
Can you also update the translation files in this PR ?

**Comment by fvanderbiest** [comment]
May I ask @severo to double check the spanish translations ? Many thanks.

**Comment by landryb** [comment]
Note that i'm doing my testing with a manually migrated user database that was still using the previous `ou=groups` hierarchy, on a dev environment, so some users might be invalid here and there... but i have groups where the member list in ldap matches what the `private/roles/XXXX` API returns, and the total member amount is sometimes off by 20 (ie 551->531). Some filtering being done (i see the `role.associate` call?) on the content of `private/users` reply ? And yes, i have `olcMemberOfRefInt:TRUE` in my slapd, so the database **should** have a consistent content.

**Comment by landryb** [comment]
Okay, i found out where the problem is, and it's a tricky one. Comparing the 'full' user list and the list actually shown in the role list, we have duplicate **valid** users in the sense that they have **several** accounts (ie `prenom.nom` and `prenom.nom1`, and sometimes `prenom.nom2`) but the LDAP entries have more or less the same givenName/sn (with differences in capitalizations, but you get the idea)

In that case, it seems the console only shown **one** entry in the user tab/list or in the role member list, while the two distinct uids are in the role.. and that might explain the differences in member total.

For example, for the role where i had 81 entries in `private/roles/XXXX` reply, in those i have two uids `admin` and `breuil` but both show up as 'Breuil Landry' and are `merged` by the UI:

```
$ldapsearch -x -h localhost -b dc=craig,dc=fr sn=BREUIL uid givenName SN

dn: uid=breuil,ou=users,dc=craig,dc=fr
uid: breuil
givenName: Landry
sn: Breuil

dn: uid=admin,ou=users,dc=craig,dc=fr
uid: admin
sn: Breuil
givenName: Landry
```

So this is imo a distinct (real, and already existing ?) issue from the one at hand here.

File: console/src/main/webapp/manager/app/assets/lang/es.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
   "tab.logs"              : "Logs",
   "tab.manage"            : "Administrar",
   "tab.area"              : "Zona de competencia",
+  "tab.users"             : "Miembro",
```
Comment (severo) [inline console/src/main/webapp/manager/app/assets/lang/es.json]: Miembros

File: console/src/main/webapp/manager/app/assets/lang/es.json [hunk 2]
```diff
@@ -96,6 +97,9 @@
   "org.select"            : "Seleccionar una entidad",
   "org.membersCount"      : "Miembros",
   "org.ohasdeleg"         : "Los usuarios de esta organización pueden ser administrados por los siguientes usuarios:",
+  "org.userremoved"       : "El usuario ha sido eliminado de la organización.",
+  "org.useradded"         : "El usuario se añadió a la organización.",
```
Comment (severo) [inline console/src/main/webapp/manager/app/assets/lang/es.json]: El usuario ha sido añadido a la organización.

File: console/src/main/webapp/manager/app/assets/lang/es.json [hunk 3]
```diff
@@ -119,6 +123,9 @@
   "role.cn"               : "Nombre",
   "role.favorite"         : "Favorite",
   "role.favorites"        : "Favorites",
+  "role.useradded"        : "El usuario se añadió al rol.",
```
Comment (severo) [inline console/src/main/webapp/manager/app/assets/lang/es.json]: El usuario ha sido añadido al rol.

**Comment by fvanderbiest** [comment]
> So this is imo a distinct (real, and already existing ?) issue from the one at hand here.

Yep. 
How do these users appear in the /console/private/users service ?

File: console/src/main/webapp/manager/app/components/role/role.tpl.html [hunk 4]
```diff
@@ -30,6 +30,53 @@ <h4>
...
+            </select>
+          </div>
+        </form>
+
+        <table class="table table-striped table-condensed">
+
+          <thead>
+            <tr>
+              <th translate>users.user</th>
+              <th translate>user.role</th>
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/components/role/role.tpl.html]: I'm not sure this is the correct translation key for a "remove" action

Comment (fvanderbiest) [reply console/src/main/webapp/manager/app/components/role/role.tpl.html]: ping @tonio

File: console/src/main/webapp/manager/app/components/role/role.tpl.html [hunk 5]
```diff
@@ -64,7 +64,7 @@ <h3 class="roles-title">
                 <a ng-link="user({id: user.uid, tab: 'infos'})">{{::user.sn}} {{::user.givenName}}</a>
               </td>
               <td>
-                <a href="javascript:void(0)" ng-click="role.associate(user.uid, true)" class="text-danger">
+                <a href="javascript:void(0)" ng-click="role.associate(user.uid, true)" class="text-danger" title="{{role.i18n.remove}}">
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/components/role/role.tpl.html]: missing role.i18n.remove and org.i18n.remove in translation files ?

**Comment by landryb** [comment]
Retested this, 👍 for me feature-wise.

**Pull request merged**